content/medo_manual/werken_met_projecten.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Werken met projecten
+sort: 200
+---
+
+* TOC
+{:toc}
+
+## Projecten
+
+In dit overzicht staan de projecten waar u toegang hebt.
+Zoekveld bovenaan helpt bij het zoeken naar een bepaald project.
+
+<img src='/assets/images/screenshots/medo/projectoverzicht.png' />
+
+## Projectgegevens
+
+In dit overzicht staan de gegevens en de instellingen van het project.
+
+<img src='/assets/images/screenshots/medo/projectpagina.png' />
+
+## Projectinstellingen
+
+Hier kunt u de instellingen van het project aanpassen. Volgende instellingen
+zijn aanpasbaar:
+
+### Titel van het project
+
+Bij voorkeur een korte en veelzeggende naam.
+
+### Verplichte velden
+
+Selecteer hier welke velden worden verplicht bij het
+[invoeren van deelnemers](/medo_manual/werken_met_deelnemers/).
+
+### Zichtbare velden
+
+Kies of alle velden van een deelnemer zichtbaar moeten zijn, of alleen de
```

Comment:
Om eerlijk te zijn: ik weet het zelf niet. Maar het is sterk gerelateerd aan https://github.com/roqua/documentation/pull/8#issuecomment-221869324